app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h2>Hello World</h2>
+<p>Adopt, don't shop!</p>
+<p>
+  The time is now: <%= Time.now %>
```

Comment:
From now on until eternity, I will come to your landing page to find the current time. Lovely.

Reply:
who needs clocks?!